src/par/CMakeLists.txt

```diff
@@ -35,6 +35,18 @@
 
 include("openroad")
 
+option(LOAD_CPLEX "Load CPLEX" ON)
+if (LOAD_CPLEX)
+  set(CMAKE_CXX_STANDARD 17)
+  set(CMAKE_MODULE_PATH               "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+  set(CMAKE_CXX_FLAGS                 "${CMAKE_CXX_FLAGS} -DIL_STD -m64 -Wall -fPIC")
+  set(CMAKE_INCLUDE_SYSTEM_FLAG_CXX   "-isystem ")
+  set(LINKER_OPTIONS                  "-Wl,--export-dynamic")
```

Comment:
Don't set these as they are global.  Use target_compile_options and target_link_options which are specific to a particular target.  

-m64 & -Wall are not necessary to set per project.